cfme/scripting/sprout.py

```diff
@@ -41,14 +41,20 @@ def main():
 @click.option('--populate-yaml', is_flag=True, default=False,
               help="Populate the yaml with the appliance")
 @click.option('--provider', default=None, help="Which provider to use")
+@click.option('--provider_type', default=None, help='A provider type to select from')
```

Comment:
here you use an underbar as a delimiter and for --template-type you use a -.    Just need to be consistent.   BTW, if you want --provider-type to use the variable provider_type, then your line would look like:

    `@click.option('--provider-type', 'provider_type', default=None, help='A provider type to select from')`

Reply:
This is described here:  http://click.pocoo.org/5/parameters/#parameter-names